stylesheets/utils/utilities.css

```diff
@@ -45,3 +45,72 @@
 .p-sticky {
   position: sticky;
 }
+
+/* Text */
+
+.text-left {
+  text-align: left;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.gray-text {
+  color: var(--shadow-color);
+}
+
+.text-sm {
+  font-size: 0.8rem;
+}
+
+.text-md {
+  font-size: 1rem;
+}
+
+.text-lg {
+  font-size: 1.5rem;
+}
+
+.text-xl {
+  font-size: 2rem;
+}
+
+.text-xxl {
+  font-size: 2.5rem;
+}
+
+/* Heading */
+h1 {
```

Comment:
you can also use line-height property for headings.

Reply:
Thanks it sounds right. But I wanted to go with default values since they have some default margins.

Reply:
Okay.